echo-pubsub-google/src/main/java/com/netflix/spinnaker/echo/pubsub/google/GooglePubsubSubscriber.java

```diff
@@ -81,7 +85,7 @@ public String subscriptionName() {
 
   @Override
   public String getName() {
-    return name;
+    return subscriptionName;
```

Comment:
We made a distinction between the `subscriptionName`, which is the actual locator in GCE and the logical `name` which is used in the UI when suggesting pub/sub subscriptions to use. Is there a reason to change that here?